pkg/signals/signal_posix.go

```diff
@@ -1,3 +1,4 @@
+//go:build !windows
```

Comment:
missing space between //
```suggestion
// go:build !windows
```

Reply:
This new build tag IS NOT supposed to have a space. This comment was automatically generated by goimports based on the previous build tag. Some more details on the build tags at https://stackoverflow.com/a/68361186.